xrpl/core/keypairs/helpers.py

```diff
@@ -15,11 +15,11 @@ def sha512_first_half(message: bytes) -> bytes:
     return hashlib.sha512(message).digest()[:32]
 
 
-def get_account_id(public_key: bytes) -> bytes:
+def get_classic_address(public_key: bytes) -> bytes:
     """
-    Returns the account ID for a given public key. See
+    Returns the classic address for a given public key. See
     https://xrpl.org/cryptographic-keys.html#account-id-and-address
-    to learn about the relationship between keys and account IDs.
+    to learn about the relationship between keys and classic addresses.
```

Comment:
This is not an exposed method so I don't think this counts as a breaking change.

Reply:
agree but not sure why you are changing the name. `account_id` has specific meaning if you look at the linked article